assets/src/dashboard/components/scrollToTop/index.js

```diff
@@ -75,19 +75,23 @@ const DropUpArrowIcon = styled(ChevronLeft)`
 const ScrollToTop = () => {
   const [isVisible, setIsVisible] = useState(false);
   const {
-    actions: { scrollToTop, addSquishListener, removeSquishListener },
+    actions: { scrollToTop },
   } = useLayoutContext();
 
-  useEffect(() => {
-    const isVisibleFromProgress = (event) => {
-      setIsVisible(event.data.progress > 0);
-    };
+  const handleScroll = useCallback(() => {
```

Comment:
Scroll events can be fired a lot. Might need to consider using `useDebouncedCallback` here.

Reply:
yup. good call